src/Components/Web.JS/src/Rendering/LogicalElements.ts

```diff
@@ -201,7 +201,8 @@ export function permuteLogicalChildren(parent: LogicalElement, permutationList:
 
   // Phase 2: insert markers
   permutationList.forEach((listEntry: PermutationListEntryWithTrackingData) => {
-    const marker = listEntry.moveToBeforeMarker = document.createComment('marker');
+    const marker = document.createComment('marker');
+    listEntry.moveToBeforeMarker = document.createComment('marker');
```

Comment:
This is a functional change as it's creating two separate comments. We should only create one.

Reply:
Fixed with latest commit!